gematria/datasets/find_accessed_addrs_exegesis.cc

```diff
@@ -0,0 +1,151 @@
+// Copyright 2023 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "gematria/datasets/find_accessed_addrs_exegesis.h"
+
+#include "BenchmarkRunner.h"
+#include "LlvmState.h"
+#include "Target.h"
+#include "TargetSelect.h"
+#include "X86.h"
+#include "X86InstrInfo.h"
+#include "X86RegisterInfo.h"
+#include "gematria/llvm/disassembler.h"
+
+using namespace llvm;
+using namespace llvm::exegesis;
```

Comment:
We should follow either the Google style guide (which disallows `using namespace` or converge on the LLVM style guide, but that would require different naming for methods in the class (and IIRC, member variables which should not end with an underscore).

I'm fine with both styles (most of the code uses the Google style now, but I think we'll need to move to the LLVM style eventually, so it would be fine to use it, as long as it is used consistently and for larger blocks of code).

Reply:
Good point. I prefer the LLVM Style, so that's mostly what I've written this in. I kept the method names with slightly different naming to be consistent with `FindAccessedAddrs`, but staying completely in the LLVM Style makes sense and I've updated the patch.

I don't think there's anything explicit about disallowing underscores within the LLVM Style guide, but I've renamed things to get rid of it as it definitely isn't canonical within LLVM.